sphinx/tutorial/rose/suites.rst

```diff
@@ -18,7 +35,7 @@ be used to do the same for a :term:`workflow`.
 Configuration Format
 --------------------
 
-A Rose suite configuration is a Cylc :term:`run directory` containing a
+A Rose suite configuration is a Cylc :term:`source directory` containing a
```

Comment:
Could be a run directory too, no?

Reply:
Yes. But...
I don't think that we want to encourage that way of working.